code/modules/power/tesla/energy_ball.dm

```diff
@@ -73,6 +70,8 @@
 				collide(E)
 
 		for (var/ball in orbiting_balls)
+			if(prob(80))  //tesla nerf/reducing lag, each miniball now has only 20% to trigger the zap
```

Comment:
just make it a define :amlooking:

Reply:
how and why would i make one small prob if into "define"?

Reply:
how and why would i change one small RNG check into define